tls/extensions/s2n_client_psk.c

```diff
@@ -166,8 +166,9 @@ static int s2n_client_psk_send(struct s2n_connection *conn, struct s2n_stuffer *
         /* Write obfuscated ticket age */
         uint32_t obfuscated_ticket_age = 0;
         uint64_t current_time = 0;
-        POSIX_ENSURE(conn->config->wall_clock(conn->config->sys_clock_ctx, &current_time) >= S2N_SUCCESS,
-                     S2N_ERR_CANCELLED);
+        
+        POSIX_GUARD_RESULT(s2n_config_wall_clock(conn->config,&current_time));
+
```

Comment:
nit: The spacing added here might be unnecessary. Same with some of the other changes.